conf/defaults.ini

```diff
@@ -1140,6 +1140,12 @@ enable =
 # The new prometheus visual query builder
 promQueryBuilder = true
 
+# The new loki visual query builder
+lokiQueryBuilder = true
```

Comment:
Is this supposed to be here?

Reply:
Thank you. you are correct. It was not there in 8.5.x branch.